Marlin/src/feature/bedlevel/ubl/ubl.h

```diff
@@ -254,7 +258,7 @@ class unified_bed_leveling {
       const int8_t cx = get_cell_index_x(rx0),
                    cy = get_cell_index_y(ry0);
 
-      if (!WITHIN(cx, 0, GRID_MAX_POINTS_X - 2) || !WITHIN(cy, 0, GRID_MAX_POINTS_Y - 2)) {
+        if (!WITHIN(cx, 0, GRID_MAX_POINTS_X - 1) || !WITHIN(cy, 0, GRID_MAX_POINTS_Y - 1)) {
```

Comment:
I edited this with a followup commit. Since the `get_cell_index_x`/`y` functions already clamp the return values, this condition will never evaluate as true.